src/utils/dateTime.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Generic function for using unified date.
+ *
+ * Long version additionally also includes the time.
+ *
+ * @param date - Date (or string representing a date) to display
+ * @param long - Whether the time should also be displayed
+ */
+export const createDateTime = (date: Date | string, long?: boolean): string =>
```

Comment:
@matedo1  I am thinking… would it not be better if we did instead two bool variables includeDate and includeTime?

Reply:
It could make the function more generic for possible future use

Reply:
yes, this sounds like a good idea, then I would also use named arguments instead of positional ones:
`createDateTime = ({date, includeDate, includeTime}) => { .. }` and both boolean values `includeDate` and `includeTime` should be `true` by default